### DISCUSSION TITLE
连接性检查，无相应。

### DISCUSSION THREAD

**Discussion by yikuaidao** [lobehub/lobehub#1237] [General Question] [answered]
你好，版本升级到最新后，不能正常回答问题，进行连接性检查，无响应，无出错提示。官方key和代理key都试过了，单独填写openai的key也不行，连接性检查无提示；进入聊天页面点发送按钮后长时间停留在问题提交状态，无反馈信息。已经重新删除镜像和容器，也进行了初始化，重新拉取后还是同样的问题，请问我怎么进行排查。
![screenshot20240206](https://github.com/lobehub/lobe-chat/assets/56807964/d762de2f-12e7-4434-982c-a95d43ce6908)

**Comment by arvinxx** [comment]
可能是这个问题：https://github.com/lobehub/lobe-chat/issues/1227#issuecomment-1926627455

**Comment by yikuaidao** [comment]
120版本使用正常，升级到123版本就不行了

**Comment by yikuaidao** [comment, marked as answer]
更新到最新的124.2都正常了
